### ISSUE TITLE
[BUG] Route names showing at bottom of all pages

### ISSUE THREAD

**Issue by eburdekin** [hackforla/expunge-assist#1625]
### What is the bug you're reporting?
The LG is showing each step's route at the bottom of the page.

### Where did you find the bug?
https://expungeassist.org/
https://expungeassist.org/welcome
https://expungeassist.org/before-you-begin

### When did you first notice this bug?
10/14 2pm

### What browser do you use?

Safari

### Do you have any screenshots or videos of the bug?
![image](https://github.com/user-attachments/assets/a6c5b4ca-b60a-43c4-800e-0cb1fabd3b5e)

### Action Items

- [ ] ... to be filled out by the Dev Lead

### Resources/Notes

REPLACE THIS TEXT - If there is documentation or tutorials that may help with this issue, provide the link(s) here.

**Comment by eburdekin** [comment]
Looks like it's because of this LocationDisplay component https://github.com/hackforla/expunge-assist/blob/c7a016eff223bb444f8de184ae38add9da04c537/products/statement-generator/src/App.tsx#L54

**Comment by whitneywind** [comment]
Yes, it's because of the location display. It is needed for the routing tests but maybe we could make it so it's not displayed visibly. I have another issue right now but I'd be happy to take this issue after!
